SU-SWS/decanter
# 202 sticky footer

# READY FOR REVIEW

# Summary
- Creates a sticky footer mixin that puts the global footer component to the bottom of the page
- Adds the global footer to the rendered style guide
- I included the work from the homepage branch in here as well so you will see that code too.

# Needed By (Date)
- ?

# Urgency
- Highish as @JBCSU needs it next week.

# Steps to Test

0. Ignore the homepage.md and other related css gunk.
1. Check out this branch
2. Compile the styleguide
3. Browse the styleguide home page
4. See the global footer on the bottom
5. Open up your browser inspector 
6. Delete the `<article class="kss-main">` element from the dom to shorten the page's content
7. See the footer still on the bottom
8. Review code.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
